src/main/java/frc/robot/components/SplitDrive.java

```diff
@@ -120,8 +120,9 @@
   private final CANSparkMax m_rightMotor;
 
   private boolean m_reported;
-  private final double m_maxOutput = 5676 * 42 / 10; // might need to multiply by 60 // Should give us the correct
-                                                     // conversion from rpm to encoder tics per 100 miliseconds
+  private final double m_maxOutput = 5676 * 42 / 10 * 60;
```

Comment:
It seems like you should divide by 60 instead, to covert from "per minute" to "per second"

Reply:
yep